qa/hooks/scripts/qa-plan-gate-lib.sh

```
@@ -92,18 +100,160 @@ qpg_stamp_valid() {
 
   echo "valid"; return 0
 }
 
+# QPG_OVERRIDE_TTL - how long a human override authorizes a branch, in seconds.
+#
+# Eight hours: one working day, so a single override covers the session it was
+# given for without becoming a permanent grant. Re-giving it costs one typed line
+# (or one terminal command), and by the time it lapses the ordinary click path is
+# almost always available again, since the usual cause of needing an override is a
+# hook that a restart has since registered.
+QPG_OVERRIDE_TTL=28800
```

Shellcheck (0.11.0)

[warning] QPG_OVERRIDE_TTL appears unused. Verify use (or export if used externally).

(SC2034)